frontend/workflows/serverexperimentation/src/start-experiment.tsx

```diff
@@ -40,7 +29,14 @@ const ExperimentDetails: React.FC<ExperimentDetailsProps> = ({
   upstreamClusterTypeSelectionEnabled,
   onStart,
 }) => {
-  const experimentDataState = useState<ExperimentData>({} as ExperimentData);
+  const initialExperimentData = {
+    faultInjectionCluster:
+      IClutch.chaos.serverexperimentation.v1.FaultInjectionCluster.FAULTINJECTIONCLUSTER_UPSTREAM,
+    faultType: FaultType.ABORT,
```

Comment:
initial state of the screen has `faulType` and `faultInjectionCluster` properties set which means that we can assume that both of these properties are always set.